.llvm-ci.yml

```diff
@@ -0,0 +1,88 @@
+resources:
+  repositories:
+  - repository: llvm
+    type: github
+    endpoint: LLVMServiceConnection
+    name: llvm/llvm-project
+
+jobs:
+- job:
+  displayName: LLVM Linux Build
+  pool: verona-llvm-build-linux
+  timeoutInMinutes: 120
+  strategy:
+    matrix:
+      Clang Release:
+        CC: clang
+        CXX: clang++
+        CXXFLAGS: -stdlib=libstdc++
+        BuildType: Release
+        BuildName: release
+        Sanitizer:
+      Clang Debug+ASAN:
+        CC: clang
+        CXX: clang++
+        CXXFLAGS: -stdlib=libstdc++
+        BuildType: Debug
+        BuildName: debug
+        Sanitizer: Address
+  steps:
+  - checkout: llvm
+
+  - script: |
+      set -eo pipefail
+      git checkout master
+      git fetch
```

Comment:
Fetch etc shouldn't be needed as that's what the checkout step does above.

Reply:
gotcha.